cpu/cpu.go

```diff
@@ -232,29 +224,33 @@ func (p *Plugin) CollectMetrics(metricTypes []plugin.MetricType) ([]plugin.Metri
 
 // GetConfigPolicy returns config policy
 // It returns error in case retrieval was not successful
-func (p *Plugin) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	cp := cpolicy.New()
-	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
-	node := cpolicy.NewPolicyNode()
-	node.Add(rule)
-	cp.Add([]string{vendor, fs, pluginName}, node)
-	return cp, nil
+func (p *Plugin) GetConfigPolicy() (plugin.ConfigPolicy, error) {
+	policy := plugin.NewConfigPolicy()
+	policy.AddNewStringRule([]string{vendor, fs, Name}, "proc_path", false, plugin.SetDefaultString("/proc"))
+
+	return *policy, nil
 }
 
-//Meta returns meta data for plugin
-func Meta() *plugin.PluginMeta {
-	return plugin.NewPluginMeta(
-		pluginName,
-		version,
-		pluginType,
-		[]string{},
-		[]string{plugin.SnapGOBContentType},
-		plugin.ConcurrencyCount(1))
```

Comment:
@rashmigottipati, this should be kept (`plugin.ConcurrencyCount(1))`) - please look how Marcin handled that in disk collector - https://github.com/intelsdi-x/snap-plugin-collector-disk/pull/47/files#diff-0e94f1a71d469b8ab3d3c4ff04bf38b4R110.